pom.xml

```diff
@@ -32,7 +32,7 @@
     <olap4j.version>1.2.0</olap4j.version>
     <eigenbase-properties.version>1.1.2</eigenbase-properties.version>
     <commons-io.version>1.4</commons-io.version>
-    <mysql-connector-java.version>5.1.39</mysql-connector-java.version>
+    <mysql-connector-java.version>8.0.27</mysql-connector-java.version>
```

Comment:
In addition, I would recommend reopening and fixing https://jira.pentaho.com/browse/MONDRIAN-2714 which solves support for the latest MySQL JDBC driver and Date values. This is our patch to fix that https://github.com/rsim/mondrian/commit/88d9dd8ce47b2c665eaafcf8c6fbf338465d90ee